service-discovery/README.adoc

```diff
@@ -31,304 +31,433 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+* You've completed the previous steps in this tutorial:
+** xref:proc-verifying-connection-to-openshift-cluster_{context}[]
+** xref:proc-connecting-kafka-instance-to-openshift-cluster_{context}[]
+* You've privileges to deploy applications in the OpenShift project created earlier in this tutorial.
 
 .Procedure
-. Execute
+
+. If you're not already logged in to the OpenShift CLI, log in using a token, as described earlier in this tutorial. Log in as the same user who created a new project earlier in the tutorial.
+
+. Use the OpenShift CLI to ensure that the current OpenShift project is the one created earlier in this tutorial, as shown in the following example.
 +
+.Using the OpenShift CLI to specify the current OpenShift project
 [source]
 ----
-rhoas cluster connect --ignore-context
+$ oc project rhoas-quarkus
+----
+
+. To deploy the Quarkus application, apply an example application template provided by {product}.
++
+.Deploying an example Quarkus application
+[source,options="nowrap"]
 ----
+$ oc apply -f https://raw.githubusercontent.com/redhat-developer/app-services-guides/main/code-examples/quarkus-kafka-quickstart/.kubernetes/kubernetes.yml
+
+service/rhoas-quarkus-kafka created
+deployment.apps/rhoas-quarkus-kafka created
+route.route.openshift.io/rhoas-quarkus-kafka created
+----
++
+As shown in the output, when you deploy the application, OpenShift creates a service and route for access to the application.
+
+. Get the URL of the route created for the application.
 +
-. You will be asked to select the Kafka instance you want to connect.
-Select instance and Press **Enter** to continue.
-. The CLI will print the **Connection Details** and asks you to confirm.
-. Verify if namespace that will be used to create service instance is `rhoas-quarkus`
-. Type `y` and press **Enter** to continue.
-. You will be asked to provide a token, which again can be retrieved from https://cloud.redhat.com/openshift/token . Navigate to this URL, copy the token to your clipboard, and copy it into your terminal. Press **Enter** to continue.
-. You should see the message: **KafkaConnection successfully installed on your cluster.**
-. To verify that the connection has been successfully created, execute the following **oc** command: `oc get KafkaConnection`. This should return a **KafkaConnection** with the name of your Kafka instance.
+.Getting the route details for the Quarkus application
+[source,options="nowrap"]
+----
+$ oc get route
+
+NAME                   HOST/PORT
+rhoas-quarkus-kafka    rhoas-quarkus-kafka-jbyrne-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com
+----
 
-.Verification
-ifdef::qs[]
-* `oc get KafkaConnection` returned created kafka instance
+. On the command line, highlight the URL shown under *HOST/PORT*. Right-click and select *Copy*.
 
-In case of issues you can remove your KafkaConnection by executing
-`oc delete akc name-of-your-kafka` and retry operation
-endif::[]
+. In your web browser, paste the URL for the route.
```

Comment:
The route is http but copy and paste a hostname without the protocol will fallback on https for most of the browsers and will only show `Application is not available`, we should specify to add `http://`

Reply:
Thanks for calling this out. I added a directive about including `http://` in the URL.